src/XRP/shared/rippled-error-messages.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * A class for creating constants that represent possible error messages received from the rippled JSON RPC.
+ */
+export default class RippledErrorMessages {
+  static accountNotFound = 'actNotFound'
+  static invalidExcludedAddress = 'invalidHotwallet'
```

Comment:
is this right? i would think it should be `invalidHotWallet` but i never know the casing with rippled

Reply:
nice catch, it is indeed that